bundle/manifests/node-healthcheck-operator.clusterserviceversion.yaml

```diff
@@ -223,15 +223,29 @@ spec:
     Remediation” API to instantiate the configured remediation template which
     specifies the mechanism/controller to be used.
 
-    ### Failure handling: Self Node Remediation
-    By default NHC depends on the “Self Node Remediation” (SNR) operator, which
-    is installed automatically.
+    ### Failure handling: External remediators
+    There are multiple remediators for handling node failure that we recommend:
+    - Self Node Remediation (SNR)
+    - Fence Agents Remediation (FAR)
+    - Machine Deletion Remediation (MDR)
+
+    By default NHC depends and uses SNR operator, which is installed automatically, as its default remediator.
```

Comment:
no, NHC has no default anymore. SNR is installed only, nothing more.

Reply:
```suggestion
    NHC is installed with the SNR operator.
```